cvrf2csaf/section_handlers/document_category.py

```diff
@@ -6,8 +6,80 @@ class DocumentCategory(SectionHandler):
     def __init__(self):
         super().__init__()
 
+        self.special_profiles = {
+            'Informational Advisory',
+            'security-incident-response',
+            'Security Advisory',
+            'veX'
+        }
```

Comment:
These values are not the ones specified for the profiles in [Section 4](https://docs.oasis-open.org/csaf/csaf/v2.0/cs01/csaf-v2.0-cs01.html#4-profiles).